structlog_pretty/processors.py

```diff
@@ -42,7 +42,9 @@ def __init__(self, digits=3, only_fields=None):
 
     def __call__(self, _, __, event_dict):
         for key, value in event_dict.items():
-            if self.only_fields is None or key in self.only_fields:
+            if ((self.only_fields is None or
+                    key in self.only_fields) and
+                    not isinstance(value, bool)):
```

Comment:
This condition got a bit out of hand here, it was already pretty bad before as well.

How about flipping these to

```python
if self.only_fields is not None and key not in self.only_fields:
    continue
if isinstance(value, bool):
    continue  # don't convert True to 1.0
```
?